doc/source/io.rst

```diff
@@ -50,6 +50,12 @@ Writing Spatial Data
 
 GeoDataFrames can be exported to many different standard formats using the ``GeoDataFrame.to_file()`` method. For a full list of supported formats, type ``import fiona; fiona.supported_drivers``.
 
+.. note::
+
+    GeoDataFrame can contain more field types than most of the file types. For example tuples or lists
```

Comment:
```suggestion
    GeoDataFrame can contain more data types than supported by most of the file formats. For example tuples or lists
```

(just a suggestion, I found the first sentence a bit difficult to understand)

Reply:
Ah, but "field type" is the term that is used in the error message?

Reply:
Yes. That is why I used field types. It is a bit of an inconsistency between python and GIS terminology. I kept field types to make a connection to fiona's error.